gstreamer-sdp/src/m_i_k_e_y_message.rs

```diff
@@ -25,17 +25,17 @@ use m_i_k_e_y_encrypt_info::MIKEYEncryptInfo;
 use m_i_k_e_y_map_s_r_t_p::MIKEYMapSRTP;
 
 impl MIKEYMessage {
-    pub fn new_from_bytes(bytes: glib::Bytes, info: MIKEYDecryptInfo) -> Result<MIKEYMessage, Error> {
+    pub fn new_from_bytes<'a, P: Into<Option<&'a MIKEYDecryptInfo>>>(bytes: &glib::Bytes, info: P) -> Result<MIKEYMessage, Error> {
         unsafe {
             let mut error = ptr::null_mut();
-            let ret = ffi::gst_mikey_message_new_from_bytes(bytes.to_glib_full(), info.to_glib_full(), &mut error);
+            let ret = ffi::gst_mikey_message_new_from_bytes(bytes.to_glib_none(), info.to_glib_full(), &mut error);
             mem::forget(bytes);
             mem::forget(info);
```

Comment:
This function does not take ownership of the bytes, same for the other one below with data.

The `info` is also not taken ownership of